### ISSUE TITLE
Operation not found with inlined OpenAPI Spec

### ISSUE THREAD

**Issue by asaintsever** [solo-io/gloo-portal-issues#5]
**Env**

- Gloo Enterprise Helm chart version: `1.5.0-beta10`
- Gloo Developer Portal Helm chart version: `0.4.8`

**Issue**

Deploy `APIDoc` below:

```
apiVersion: devportal.solo.io/v1alpha1
kind: APIDoc
metadata:
  name: echoserver-schema
spec:
  openApi:
    content:
      inlineString: |
        {
          "openapi": "3.0.1",
          "info": {
            "title": "Echoserver",
            "description": "Simple server that responds with the http headers it received",
            "version": "1.0"
          },
          "paths": {
            "/": {
              "get": {
                "summary": "Get",
                "operationId": "get",
                "responses": {
                  "200": {
                    "description": null
                  }
                }
              }
            }
          }
        }
```

Gloo Developer Portal displays the Echoserver API but fails to find the GET operation ("There are no Operations to display!"). Clicking on the `View API Spec` button properly shows the OAS and the operation.

**Comment by tcjiang9** [comment]
Was able to reproduce on version 0.4.8 Using
`helm install dev-portal dev-portal/dev-portal -n dev-portal --values gloo-values.yaml --version v0.4.8`
and applying the problematic API Doc
However, on the latest version, it looks like this has been fixed, use
`helm install dev-portal dev-portal/dev-portal -n dev-portal --values gloo-values.yaml --version v0.5.0`
and apply the problematic API Doc

**Comment by tcjiang9** [comment]
@asaintsever closing as fixed in latest version, please update to v0.5.0
